testsuite/pytests/sli2py_mpi/test_self_get_conns_with_empty_ranks.py

```diff
@@ -19,13 +19,12 @@
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
-import numpy as np
-import pandas
```

Comment:
Was this intentional?

Reply:
Yes, the imports are not needed here. I just pushed another commit that removes superfluous imports in other tests as well, removes old-style GSL-checking and adds comments on data generated to all test docstrings.